wrapper/CSharp/wolfSSL-Example-IOCallbacks/wolfSSL-Example-IOCallbacks.cs

```diff
@@ -191,30 +253,7 @@ static void Main(string[] args)
             return;
         }
 
-        StringBuilder ciphers = new StringBuilder(new String(' ', 4096));

```

Comment:
Guessing the PSK code was a copy/paste from the PSK example and doesn't belong here?

Reply:
The static PSK example is not needed for IO callbacks and it conflicts with showing certificate verification callback since using static cipher suites. Wanted to keep all the callback examples in one location so I removed the PSK use in this example, there is other examples in the directory that still have PSK for users to see.